packages/yoga/src/Tag/native/Informative.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { func, oneOfType, oneOf, bool, node } from 'prop-types';
+import { hexToRgb } from '@gympass/yoga-common';
+import { StyledTag, StyledText } from './Tag';
+
+const Informative = styled(StyledTag)`
+  ${({
+    variant,
+    theme: {
+      yoga: {
+        colors: {
+          text,
+          feedback: {
+            [variant]: { light = text.secondary },
+          },
+        },
+        components: { tag },
+      },
+    },
+  }) => `
+    background-color: ${hexToRgb(light)};
```

Comment:
```suggestion
    background-color: ${light};
```

Do we need this `hexToRgb`?